### PR TITLE
Home page tweaks

### PR DESCRIPTION
**Pull request by JasonChong96**
## Problem

The home page implementation differs from the current design.

Closes [#89]

## Solution

The home page has been tweaked to match the design (other than file upload related elements).

## Improvements
Analytics icon should load correctly now

## Before & After Screenshots

**BEFORE**:
PC:
[Image 1](https://user-images.githubusercontent.com/35889982/82210645-080a3780-9942-11ea-8e66-b0a410e044e4.png)

Samsung Note 9:
[Image 1](https://user-images.githubusercontent.com/35889982/82210746-3851d600-9942-11ea-8d64-ead426314c2f.png)

**AFTER**:
PC:
[Image 1](https://user-images.githubusercontent.com/35889982/82205908-e311c680-9939-11ea-9653-d4fbe19f24e4.png)

Phone X (Simulated):
[Image 1](https://user-images.githubusercontent.com/35889982/82206476-e2c5fb00-993a-11ea-8700-e5eaf959ad67.png)
[Image 2](https://user-images.githubusercontent.com/35889982/82206521-f8d3bb80-993a-11ea-8f76-c95003d6ab6f.png)

iPad Pro (Simulated):
[Image 1](https://user-images.githubusercontent.com/35889982/82206558-0ab55e80-993b-11ea-8eba-079dcef0e93b.png)

Samsung Note 9:
[Image 1](https://user-images.githubusercontent.com/35889982/82208902-f1161600-993e-11ea-8759-1ee0f411e11e.png)

### CODE REVIEW

File: src/client/theme/index.js [hunk 1]
```diff
@@ -34,22 +34,22 @@ export default responsiveFontSizes(
...
+        fontWeight: '500',
       },
       h2: {
-        fontSize: '1.5rem',
+        fontSize: '1.75rem',
         fontWeight: '600',
       },
       h3: {
-        fontSize: '1.2rem',
+        fontSize: '1.375rem',
```
Comment (kylerwsm) [inline src/client/theme/index.js:52]: I believe our text styles are pretty outdated. Can you help me update this to the latest values from [here](https://app.zeplin.io/project/5c4770b601c52637747d1840/styleguide/textstyles), and make relevant changes to the existing components? 😁

File: src/client/components/HomePage/StatisticsGraphic/index.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      maxWidth: '100%',
+    },
+  }),
+)
+
+const StatisticsGraphic = () => {
+  const classes = useStyles()
+
+  return (
+    <img className={classes.root} src={statsGraphic} alt="Statistics graphic" />
```
Comment (kylerwsm) [inline src/client/components/HomePage/StatisticsGraphic/index.jsx:17]: In the `img` tag, can we add a `draggable="false"`, or a `userDrag="false"` in the jss to make the image not draggable?

File: src/client/components/HomePage/FeatureListSliver.jsx [hunk 3]
```diff
@@ -4,15 +4,16 @@ import {
...
   makeStyles,
   useMediaQuery,
   useTheme,
 } from '@material-ui/core'
-import antiPhisingIcon from '~/assets/icons/anti-phishing-icon.svg'
-import customisedIcon from '~/assets/icons/customised-icon.svg'
-import analyticsIcon from '~/assets/icons/analytics-icon.svg'
+import antiPhisingIcon from '~/assets/icons/home-page-anti-phishing-icon.svg'
+import customisedIcon from '~/assets/icons/home-page-customised-icon.svg'
+import analyticsIcon from '~/assets/icons/home-page-analytics-icon.svg'
```
Comment (kylerwsm) [inline src/client/components/HomePage/FeatureListSliver.jsx:16]: This looks good

**Comment by JasonChong96** [comment]
> Added some comments. At the same time, given that we have quite a lot of graphical assets now, what are your opinions on encapsulating assets into the relevant components?
I think its a good idea since some of the component files are getting rather long

File: src/client/theme/index.js (hunk 1 above)
Comment (JasonChong96) [reply src/client/theme/index.js]: Updated

**Comment by JasonChong96** [comment]
> Changes look good. But I think there are some changes to be done for the font sizes after the theme changes, such as the sign-in text ([before](https://user-images.githubusercontent.com/28863045/82357574-e3908700-9a37-11ea-8d8c-20152c43bb10.png), [after](https://user-images.githubusercontent.com/28863045/82357777-28b4b900-9a38-11ea-87bf-9b8dcad179aa.png)) and the footer. Might be good to run through the rest of the pages as well to see if everything looks fine.
> 
> And thanks for working on this, it's really tedious work!

Resolved the sign-in text issue. What is the issue with the footer?

**Pull request merged**